TTSJSON/ftc_base.json

```diff
@@ -24153,9 +24153,9 @@
       "HideWhenFaceDown": false,
       "Hands": false,
       "CustomPDF": {
-        "PDFUrl": "https://www.warhammer-community.com/wp-content/uploads/2023/06/htkssr6vgkoqDAJn.pdf",
+        "PDFUrl": "https://assets.warhammer-community.com/warhammer40000_combatpatrol_rules_eng.24.09-rbtns7zwbh.pdf",
         "PDFPassword": "",
-        "PDFPage": 0,
+        "PDFPage": 6,
```

Comment:
I specifically had the combat patrol pdf open to the missions, but I can keep it on page 0 instead.